tests/Mono.Android-Tests/Mono.Android-Tests.csproj

```diff
@@ -29,6 +29,7 @@
     <_ASANScript>..\..\build-tools\wrap.sh\asan.sh</_ASANScript>
     <_UBSANScript>..\..\build-tools\wrap.sh\ubsan.sh</_UBSANScript>
     <_MonoAndroidTestPackage>Mono.Android_Tests</_MonoAndroidTestPackage>
+    <AndroidUseIntermediateDesignerFile>True</AndroidUseIntermediateDesignerFile>
```

Comment:
Can we rely on the default values in these test projects? .NET 8+ will have this turned on, and classic will have it off?